ada_feeding/config/ada_feeding_action_servers.yaml

```diff
@@ -2,7 +2,7 @@
 ada_feeding_action_servers:
   ros__parameters:
     # If we haven't received a message from the watchdog in this time, stop all robot motions
-    watchdog_timeout_sec: 0.5
+    watchdog_timeout: 0.5
```

Comment:
If not in the variagble name, add `# seconds` or `# s` afterwards to specify unit.

Reply:
Agreed. I've increasingly been adding the units to the end of the parameter name because if people define a custom config file, then the comment might go away. But I haven't yet uniformly enforced that covention.